cirq-core/cirq/ops/three_qubit_gates.py

```diff
@@ -117,8 +118,13 @@ def _decompose_(self, qubits):
 
         p = common_gates.T**self._exponent
         sweep_abc = [common_gates.CNOT(a, b), common_gates.CNOT(b, c)]
-
-        return [
+        global_phase = (1j) ** (2 * self.global_shift * self._exponent)
```

Comment:
nit: no need for parens around `1j`.  or you can make it consistent with below (line 340) and add parens to both if that is more readable.

Reply:
done